.gitignore

```diff
@@ -63,3 +63,16 @@ build/
 .cache/
 zephyr-windows.code-workspace
 flash.bin
+app_grim_reefer/extra/data/all.cap
+app_grim_reefer/extra/out/adc.csv
+app_grim_reefer/extra/out/fast.csv
+app_grim_reefer/extra/out/pre_alt.csv
+app_grim_reefer/extra/out/pre_imu.csv
+app_grim_reefer/extra/out/slow.csv
+app_grim_reefer/extra/data/lall2.cap
+app_grim_reefer/extra/out2/adc.csv
+app_grim_reefer/extra/out2/fast.csv
+app_grim_reefer/extra/out2/pre_alt.csv
+app_grim_reefer/extra/out2/pre_imu.csv
+app_grim_reefer/extra/out2/slow.csv
+app_grim_reefer/extra/out2/combined.csv
```

Comment:
Can we just wildcard everything in the extra folder or like out* and data

Reply:
yeah lol, got lazy and selected all > add to gitignore in vscode. Will wildcard